types/hammerjs/index.d.ts

```diff
@@ -210,6 +216,12 @@ declare class HammerInput
   /** Array with all new/moved/lost pointers. */
   changedPointers:any[];
 
+  /** Maximum number of pointers detected in the gesture */
+  maxPointers:number;
+
+  /** Timestamp of a gesture */
+  timeStamp:number;
```

Comment:
Isn't this a `string`? Like in `Date()`?
https://github.com/hammerjs/hammer.js/blob/ff687ea0daa3c806b9accd2ecb1a46165ea3c00a/src/utils/utils-consts.js#L7

Reply:
I was judging by my `console.log(event)` which returns:
![image](https://user-images.githubusercontent.com/12465392/104434728-d5c1e000-558b-11eb-9201-29a8225fe507.png)

I ran hammer.js tests that also confirms this is a number:
![image](https://user-images.githubusercontent.com/12465392/104435445-af507480-558c-11eb-8eec-959c100fd393.png)

Reply:
cool, thx for checking this!